service/src/test/kotlin/no/nav/su/se/bakover/service/vedtak/VedtakServiceImplTest.kt

```diff
@@ -161,6 +169,59 @@ internal class VedtakServiceImplTest {
         ).kopierGjeldendeVedtaksdata(UUID.randomUUID(), LocalDate.EPOCH.plusDays(7)) shouldBeLeft KunneIkkeKopiereGjeldendeVedtaksdata.UgyldigPeriode(Periode.UgyldigPeriode.FraOgMedDatoMåVæreFørsteDagIMåneden)
     }
 
+    @Test
+    fun `henter tidligere informasjon for overleppende vedtak`() {
+        val vedtak1 = vedtakSøknadsbehandlingIverksattInnvilget(Stønadsperiode.create(Periode.create(1.januar(2021), 31.desember(2021)))).copy(
+            opprettet = Tidspunkt.now(fixedClock)
+        )
+        val vedtak2 = vedtakSøknadsbehandlingIverksattInnvilget(Stønadsperiode.create(Periode.create(1.januar(2021), 31.desember(2021)))).copy(
```

Comment:
Det er vel egentlig bare vedtak1 som bør være et søknadsbehandlingsvedtak :shrug:

Reply:
Hehe ja, i praksis så är det så. Men skal ikke være någon forskjell i testen så länge man har en rekkefølge med forskjellige vedtak. Prøvde å få till det med revurderinger men blev väldigt mycket styr 🙈  Må utvide den Testdata-klassen for revurdering.